misc/start_faux

```diff
@@ -72,8 +72,8 @@ fi
 
 if [ -n "${options[bacnet]}" ]; then
     echo Starting bacnet loop device.
-    java -cp bacnet4j/bacnet4j-1.0-SNAPSHOT-all.jar \
-         com.serotonin.bacnet4j.test.LoopDevice $local_ip $broadcast_ip &
+    java -cp bacnet4j/bacnet4j.jar \
+	 com.serotonin.bacnet4j.test.DaqTest.BacnetDevicesServer.DistechServer $local_ip $broadcast_ip &
```

Comment:
Is Distech a specific company?  If so, we shouldn't use that here and need to make it generic/anonymous.

Reply:
Yes, Distech is the device name we are using in our test rig. The reason why I created my own object(DistechServer) - instead of using the LoopDevice object - was to emulate a faux device as similar as the Distech and test the "bacnet pic test" module I was working on. However, after a conversation with Francesco, I'm now waiting on having more clarification about the implementation of this test as - as you said - this module should be as generic as possible. I acknowledge that this should not be in this pull request.